flaskfinalproject.code-workspace

```diff
@@ -0,0 +1,10 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"python.pythonPath": "venv/bin/python3"
+	}
+}
```

Comment:
@PKCRY if you are going to use an editor that drops files like this in your projects, you should add these files to your global .gitignore file so you stop forcing them onto all of your coworkers. 😂